cmd/api/database_test.go

```diff
@@ -97,18 +101,36 @@ func TestListBooks(t *testing.T) {
 		is := is.New(t)
 
 		// Write the List Books test here.
-		// ...
-		is.Fail()
+		returnedBooks, err := listBooks()
+		is.NoErr(err)
+		is.Equal(returnedBooks, []Book{})
 	})
 
 	t.Run("List books without errors", func(t *testing.T) {
 		is := is.New(t)
 
 		// Setting up, creating books to be listed.
-		// ...
+		var bookslist []Book
+		listSize := 100
```

Comment:
I like the spirit of this, but there is no need to create 100 books to do the test. It's essentially the same as testing with 11.

Let's keep it around 11 or 20.

Reply:
Test should run fast.